languages/common-lisp/bin/template/.meta/example.lisp

```diff
@@ -0,0 +1,6 @@
+(defpackage <SLUG>
```

Comment:
`#:<SLUG>` (or just `:<SLUG>`) here...

Reply:
I've seen it done in a couple of ways, is the advantage of a keyword the guarantee that no other symbol's value is overwritten?

I don't really mind what we end up using here (as long as it isn't a string)

Reply:
I can't seem to keep consistent myself :) There is probably not a big difference. Using a :keyword means it will not overwrite anything else, using an uninterned #:keyword perhaps saves a tiny bit of memory.